### PR TITLE
Pam 393 vale updates design

### PR DESCRIPTION
**Pull request by pamgoodrich**
- Ran all design topics through Vale. 
- Added some words to the accept list and alphabetized the list for ongoing ease of use.
- Added a comment in a couple of topics where the UI term violated Vale rules.
- Left a few things that we should add to Vale at some point. Most notably, the code types in this topic:
https://meta.stoplight.io/docs/platform/ZG9jOjM2OTM3Mjkz-get-request-samples

### CODE REVIEW

File: 3.-design/api-security.md [hunk 1]
```diff
@@ -2,116 +2,132 @@
...
-For OAuth 2, select the _oauth2_ scheme type. OAuth is an open authentication standard defined by [RFC6749](https://tools.ietf.org/html/rfc6749) and [RFC6750](https://tools.ietf.org/html/rfc6750). OAuth also uses the standard HTTP _Authorization_ header to send a credential known as bearer token but also defines various flows for retrieving this bearer token. You can enter the configuration for the flows into your OpenAPI specification, and various OpenAPI tools can use this information to assist the user in going through them.
+To create an API key:
 
-Hint: If you are documenting an API that still uses OAuth 1, select the _http_ scheme type and then _oauth_ as a subtype instead.
+1. Select the **apiKey** scheme type. 
+2. Use the right menu to toggle between **query**, **header**, and **cookie**.
+3. Add a name for the query, cookie, or header name.
 
-![Oauth Configuration](https://stoplight.io/api/v1/projects/cHJqOjI/images/1TXQEE6KFfY)
+### OAuth2
```
Comment (heitortsergent) [inline 3.-design/api-security.md:44]: I think it's more common to have space between, like "OAuth 2".

File: 3.-design/api-security.md [hunk 2]
```diff
@@ -2,116 +2,132 @@
...
+- Most APIs use OAuth because authorized API calls need a user context. For this purpose,  **authorizationCode** flow is the most common OAuth flow.
+- If an application consuming the API is only a browser-based application, you may want to support the **implicit** flow. However, there are OAuth extensions like PKCE that assist in implementing the more secure **authorizationCode**_ flow for these applications, so you may not need **implicit**.
+- If your application uses OAuth for API calls without a user context, you can add **clientCredentials**.
+- Don't use the **password** flow unless you understand the security implications.
 
 You can learn more about these flows on the [OAuth website](https://oauth.net/).
 
-A button for OAuth 2 Flow Settings is located next to the input. Click it to open a popup where you can enter additional information for the selected flows:
+> **Hint:**
+> If your API uses OAuth1, select the **http** scheme type, and then select **oauth** as a subtype.
```
Comment (heitortsergent) [inline 3.-design/api-security.md:58]: Same as above comment, I think "OAuth 1" is more common.

File: 3.-design/api-security.md [hunk 3]
```diff
@@ -2,116 +2,132 @@
...
 
-- Most APIs use OAuth because authorized API calls need a user context. For this purpose, you typically use the _authorizationCode_ flow. It is the most common OAuth flow.
-- If an application consuming the API is a browser-based application without a backend, you may want to support the _implicit_ flow. However there are OAuth extensions like PKCE that assist in implementing the more secure _authorizationCode_ flow for these applications as well, so you may not need _implicit_.
-- If your application uses OAuth for API calls without a user context, you can add _clientCredentials_.
-- Do not use the _password_ flow unless you know exactly what you are doing!
+- Most APIs use OAuth because authorized API calls need a user context. For this purpose,  **authorizationCode** flow is the most common OAuth flow.
+- If an application consuming the API is only a browser-based application, you may want to support the **implicit** flow. However, there are OAuth extensions like PKCE that assist in implementing the more secure **authorizationCode**_ flow for these applications, so you may not need **implicit**.
```
Comment (heitortsergent) [inline 3.-design/api-security.md:51]: I would keep the previous way it was written, "browser-based application without a backend". I don't think "only a browser-based application" has the same meaning.

File: 3.-design/api-security.md [hunk 4]
```diff
@@ -2,116 +2,132 @@
...
+> **Hint:**
+> If your API uses OAuth1, select the **http** scheme type, and then select **oauth** as a subtype.
 
-- For _implicit_, you can provide Refresh URL and Authorization URL.
-- For _password_ and _clientCredentials_, you can provide Refresh URL and Token URL.
-- For _authorizationCode_, you can provide Refresh URL, Authorization URL, and Token URL.
+![OAuth Configuration](https://stoplight.io/api/v1/projects/cHJqOjI/images/1TXQEE6KFfY)
 
-The Refresh URL is only required if your OAuth server issues refresh tokens.
+To create an oath2 key:
```
Comment (heitortsergent) [inline 3.-design/api-security.md:62]: Same comment as above, "OAuth 2".

File: 3.-design/api-security.md [hunk 5]
```diff
@@ -2,116 +2,132 @@
...
+While OpenAPI supports the different variants of HTTP authentication, the majority of APIs use **basic**. With basic authentication, an API client sends the combination of a username and a password in the Authorization header. The method for retrieving the username and password isn't part of the OpenAPI description.
 
 ![HTTP Auth Configuration](https://stoplight.io/api/v1/projects/cHJqOjI/images/mYOOyhiBk4Q)
 
-Use the right dropdown to switch between the different HTTP authentication schemes. OpenAPI and, by extension, Stoplight support all the [schemes registered at IANA](https://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml). Further configuration is neither necessary nor possible.
+To create an HTTP key:
 
-While OpenAPI supports the different variants of HTTP authentication, the majority of APIs use _basic_. With basic authentication, an API client sends the combination of a username and a password in the Authorization header. The method for retrieving the username and password is not part of the OpenAPI description.
+1. For HTTP authentication, select the **http** scheme type.
+2. Select an HTTP authentication schemes.
```
Comment (heitortsergent) [inline 3.-design/api-security.md:98]: Should it be "scheme" here?

File: 3.-design/api-security.md [hunk 6]
```diff
@@ -2,116 +2,132 @@
...
 
-## Specifying Global Security
+## Specify Global Security
 
-You can specify global security requirements in the API Overview, right below the definition of the security schemes. The following applies for global security requirements:
+You can specify global security requirements in the API Overview. Use these global security requirements:
```
Comment (heitortsergent) [inline 3.-design/api-security.md:102]: I think this is changing the meaning of the sentence, maybe it's best to keep "The following applies for"?

Comment (pamgoodrich) [reply 3.-design/api-security.md:102]: I changed it to "Follow these global security requirements."

File: 3.-design/api-security.md (hunk 4 above)
Comment (pamgoodrich) [reply 3.-design/api-security.md:62]: Done

File: 3.-design/api-security.md (hunk 3 above)
Comment (pamgoodrich) [reply 3.-design/api-security.md:51]: Agree and fixed.

File: 3.-design/setting-up-a-mock-server.md [hunk 7]
```diff
@@ -73,7 +73,7 @@ paths:
       # ...
 ```
 
-Swap that URL for your hosted Prism URL:
+Swap that URL of your hosted Prism URL:
```
[the remark this answers is not in the document]
Comment (pamgoodrich) [reply 3.-design/setting-up-a-mock-server.md:76]: Done

File: 3.-design/design-overview.md [hunk 8]
```diff
@@ -4,28 +4,26 @@ tags: []
...
 
-- Publish all or part of your API
-- Send requests to your API to debug it
-- Create a mock API based on your descriptions
-- Validate your files with Spectral, our open-source API description linter
-- ...and much more
+- Publish all or part of your API. See [Types of Documentation](../4.-documentation/b.types-of-documentation.md).
+- Send requests to your API to debug it. See [Try It](try-it.md).
+- Create a mock API based on your descriptions. See [Mock Servers](setting-up-a-mock-server.md)
+- Validate your files with Spectral, the open-source API description linter
```
Comment (heitortsergent) [inline 3.-design/design-overview.md:23]: This bullet point and the one above are missing a "." at the end for consistency.

File: 3.-design/http-endpoints.md [hunk 9]
```diff
@@ -4,19 +4,18 @@ tags: []
 
 # Work with Models
 
-Stoplight's schema editor allows you to easily design and model data structures
-used by your API. The schema editor is especially useful for:
+Stoplight's Schema Editor allows you to design and model data structures
+used by your API. The Schema Editor is especially useful for:
```
Comment (heitortsergent) [inline 3.-design/http-endpoints.md:8]: I noticed some files have some odd line breaks in the paragraphs... It doesn't affect how it displays after it's rendered, but I've been fixing them when I see it. Looks like this is one of them.

**Review by heitortsergent — APPROVED** [review]
Left some comments but it's just small things. 👍

**Pull request merged**